rpms/base.pan

```diff
@@ -2,16 +2,49 @@ unique template rpms/base;
 
 variable OS_CORE_ONLY ?= false;
 variable OS_CORE_ISCSI_ENABLED ?= false;
+variable OS_CORE_RDMA_ENABLED ?= false;
+
 
 # Standard YUM groups for all machines
-include {'rpms/group/core' };
-include { if ( ! OS_CORE_ONLY ) 'rpms/group/base' };
-include { if ( ! OS_CORE_ONLY ) 'rpms/management-utils' };
+
+'/software/groups' = {
+  # Default packages are mainly firmware for Wifi
+  SELF['core'] = dict('mandatory', true,
+                      'default',false,
+                      'optional', false,
+                     );
```

Comment:
I'm not sure it is consistent, I can find plenty of examples of both looking around the template libraries, but yes we should agree on a standard.